include/model.h

```diff
@@ -72,6 +76,7 @@ class Model{
     std::unordered_map<std::string, double> radius_map;
     std::unordered_map<std::string, int> elem_Z;
     std::map<std::string, int> atom_amounts;
+    std::map<std::string, int> unit_cell_atom_amounts; // stores atoms of unit cell to generate chemical formula

```

Comment:
What is the difference between these? The app only allows you to either input a structure or a unit cell so in each calculation one map will remain unused? Why not just use a single map for both cases